doc/users/dflt_style_changes.rst

```diff
@@ -446,6 +446,39 @@ The behavior of the PS and Agg backends was DPI dependent, thus::
 
 There is no API level control of the hath linewidth.
 
+Fonts
+=====
+
+The default font has changed from "Bitstream Vera Sans" to "DejaVu
+Sans".  "DejaVu Sans" is an improvement on "Bistream Vera Sans" that
+adds more international, and math characters, but otherwise has the
+same appearance.  Latin, Greek, Cyrillic, Armenian, Georgian, Hebrew,
+and Arabic are `all supported
+<http://dejavu-fonts.org/wiki/Main_Page>`__ (but right-to-left render
+is still not handled by matplotlib).  In addition to the practical,
+DejaVu contains a sub-set of emoji support
+
+.. plot::
+
+   import matplotlib.pyplot as plt
+
+   fig, ax = plt.subplots()
+   tick_labels = ['😃', '😎', '😴', '😲', '😻']
+   ax.bar(range(5), [1, 4, 9, 16, 25],
+          tick_label=tick_labels, align='center')
+   ax.xaxis.set_tick_params(labelsize=20)
+   ax.set_title('Участок под застройку с смайликам')
```

Comment:
I put 'plot with emoji' into google translate and that is what came out.  Getting Cyrillic to render is a common question.  The goal here was to show it works by default now.